README.md

```diff
@@ -10,67 +10,110 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-#### Local setup:
+#### Local setup
+
 Need `mysql` or `mariadb` installed.
```

Comment:
Why `mysql` or `mariadb`?

Reply:
It's for local setup. Forgot to remove it. We're having kubernetes setup since now.